src/test/java/spoon/reflect/ast/AstCheckerTest.java

```diff
@@ -57,6 +59,22 @@
 
 public class AstCheckerTest {
 
+	@Test
```

Comment:
You could use the `@ModelTest` annotation here instead. This way, you don't need the arrange block.
There are a few examples already you can look into if needed.